packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -66,7 +68,9 @@ module.exports = function(config) {
 					result = result.default();
 				}
 				if (typeof result === 'function') {
-					result = result();
+					console.log(yellow(`Fetching URLs from ${config.prerenderUrls}`));
+					result = await result();
+					console.log(yellow(`Fetched URLs from ${config.prerenderUrls}`));
```

Comment:
shouldn't this use our log functions from util?

Reply:
done